gma/src/include/firebase/gma/internal/README.md

```diff
@@ -0,0 +1,4 @@
+# Firebase C++ Open Source Development
+
+This folder contains the header files that are currently used only for internal usage.
+Users shouldn't expect any stability guarantees for code in this folder, and any public usage is discouraged.
```

Comment:
Maybe we should be more explicit here that these methods should not be used and they will not work.

Reply:
Makes sense. Changed it to explicitly specify that these methods should not be used.